compatibility_server/compatibility_checker_server.py

```diff
@@ -71,30 +74,16 @@ def _parse_python_version_to_interpreter_mapping(s):
 
 class CompatibilityServer:
 
-    def __init__(self, host: str, port: int, clean: bool,
-                 python_version_to_interpreter: typing.Mapping[str, str],
-                 install_once: bool):
+    def __init__(self, host: str, port: int):
         """Initialize an HTTP server that checks for pip package compatibility.
 
         Args:
             host: The host name to listen on e.g. "localhost".
             port: The port number to listen on e.g. 80.
-            clean: If True then uninstall previously installed packages before
-                handling each request.
-            python_version_to_interpreter: Maps python version e.g. "3" to
-                a Python interpreter that can corresponds to that version e.g.
-                "/usr/bin/python3.6"
-            install_once: If True then the server will exit after handling a
-                single request that involves installing pip packages.
         """
         self._host = host
         self._port = port
-        self._clean = clean
-        self._python_version_to_interpreter = python_version_to_interpreter
-        self._install_once = install_once
-
-    def _shutdown(self):
-        threading.Thread(target=self._httpd.shutdown).start()
+        self._python_version_to_interpreter = PYTHON_VERSION_TO_INTERPRETER
```

Comment:
Do you need self. _python_version_to_interpreter ?

Reply:
Removed.